tools/restatectl/src/commands/metadata/get.rs

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2024 -  Restate Software, Inc., Restate GmbH.
+// All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use bytestring::ByteString;
+use clap::Parser;
+use cling::{Collect, Run};
+use tracing::debug;
+
+use restate_rocksdb::RocksDbManager;
+use restate_types::config::Configuration;
+use restate_types::live::Live;
+
+use crate::commands::metadata::{
+    create_metadata_store_client, GenericMetadataValue, MetadataAccessMode, MetadataCommonOpts,
+};
+use crate::environment::metadata_store;
+use crate::environment::task_center::run_in_task_center;
+
+#[derive(Run, Parser, Collect, Clone, Debug)]
+#[clap()]
+#[cling(run = "get_value")]
+pub struct GetValueOpts {
+    #[clap(flatten)]
+    metadata: MetadataCommonOpts,
+
+    /// The key to get
+    #[arg(short, long)]
+    key: String,
+}
+
+async fn get_value(opts: &GetValueOpts) -> anyhow::Result<()> {
+    let value = match opts.metadata.access_mode {
+        MetadataAccessMode::Remote => get_value_remote(opts).await?,
+        MetadataAccessMode::Direct => get_value_direct(opts).await?,
+    };
+
+    let value = serde_json::to_string_pretty(&value).map_err(|e| anyhow::anyhow!(e))?;
```

Comment:
This can simply be
```rust
let value = serde_json::to_string_pretty(&value)?;
```
when returning `anyhow::Result` it automatically creates the anyhow error from any error

Reply:
Totally missed your comment! I knew this but didn't see the opportunity, will fix on the next iteration if you don't beat me to it.